package.json

```diff
@@ -44,6 +44,7 @@
     "intern": "~3.2.1",
     "istanbul": "0.4.3",
     "remap-istanbul": ">=0.6.4",
+    "sinon": "1.14.1",
```

Comment:
I fixed at the same version as `dojo/core`. Will bump to `^1.17.4` as long as it plays well.